### PR TITLE
Add websockets to Inventory Source Details

### PR DESCRIPTION
**Pull request by nixocio**
Add websockets to Inventory Source Details

image

See: https://github.com/ansible/awx/issues/9013

### CODE REVIEW

**Review by AlexSCorey — COMMENTED** [review]
On the projects page, when we create a new project and click save a sync is started as we move to the details view, and that change is reflected in the sync status icon.    Seems like something we should implement here.  Also, I'd expect the name of the inventory source to always be the first thing displayed on the details view instead of the sync status icon being displayed first.

File: awx/ui/src/screens/Inventory/InventorySourceDetail/InventorySourceDetail.js [hunk 1]
```diff
@@ -226,16 +275,26 @@ function InventorySourceDetail({ inventorySource }) {
...
+            <InventorySourceSyncButton source={inventorySource} icon={false} />
+          ))}
         {user_capabilities?.delete && (
           <DeleteButton
             name={name}
             modalTitle={t`Delete inventory source`}
             onConfirm={handleDelete}
             deleteDetailsRequests={deleteDetailsRequests}
             deleteMessage={t`This inventory source is currently being used by other resources that rely on it. Are you sure you want to delete it?`}
+            isDisabled={job?.status === 'running'}
```
Comment (mabashian) [inline awx/ui/src/screens/Inventory/InventorySourceDetail/InventorySourceDetail.js:297]: I can't remember - can you delete jobs that are in `new`, `pending`, or `waiting`?

**Review by mabashian — APPROVED** [review]
Code looks good to me - I think this is a big improvement and great for consistency.

File: awx/ui/src/screens/Inventory/InventorySourceDetail/InventorySourceDetail.js (hunk 1 above)
Comment (nixocio) [reply awx/ui/src/screens/Inventory/InventorySourceDetail/InventorySourceDetail.js:297]: This change will disable the possibility to delete the inventory source when the sync is running.  Same pattern that we have for Project Details. 

The one to cancel the job itself is here - https://github.com/ansible/awx/blob/edceb0274291f3095a0b2e53bf20bd1256defa0d/awx/ui/src/screens/Inventory/InventorySourceDetail/InventorySourceDetail.js#L279

**Comment by nixocio** [comment]
> On the projects page, when we create a new project and click save a sync is started as we move to the details view, and that change is reflected in the sync status icon. Seems like something we should implement here. Also, I'd expect the name of the inventory source to always be the first thing displayed on the details view instead of the sync status icon being displayed first.

I updated the name location.  I also looked at the previous UI how the sync of the inventory source was implemented. It does not automatically sync like project does, we can file an issue for a future enhancement.

**Review by AlexSCorey — APPROVED** [review]

**Pull request merged**